master/buildbot/test/unit/test_reporter_gerrit.py

```diff
@@ -489,3 +493,20 @@ def testBuildGerritCommand(self):
             ['ssh', 'user@serv', '-p', '29418', 'gerrit', 'review', '--project project', '--notify OWNER',
              "--message 'bla'", '--verified 1', 'revision'],
             env=None)
+
+    @defer.inlineCallbacks
+    def test_callWithVersion_bytes_output(self):
+        gsp = yield self.setupGerritStatusPushSimple()
+        exp_argv = ['ssh', 'user@serv', '-p', '29418', 'gerrit', 'version']
+
+        def spawnProcess(pp, cmd, argv, env):
+            self.assertEqual([cmd, argv], [exp_argv[0], exp_argv])
+            pp.errReceived(b'test stderr\n')
+            pp.outReceived(b'gerrit version 2.14\n')
+            pp.outReceived(b'(garbage that should not cause a crash)\n')
+            so = error.ProcessDone(None)
+            pp.processEnded(failure.Failure(so))
+        self.patch(reactor, 'spawnProcess', spawnProcess)
+        # FIXME version should not contain 2.14, that is a bug.
```

Comment:
should not contain '\n'

I think it is safe to add the needed strip in that patch
lets dont do be over procedural.